tests/rptest/tests/archival_test.py

```diff
@@ -203,14 +203,26 @@ def test_write(self):
         self.kafka_tools.produce(self.topic, 10000, 1024)
         validate(self._quick_verify, self.logger, 90)
 
-    @ignore  # https://github.com/vectorizedio/redpanda/issues/1977
     @cluster(num_nodes=3)
     def test_isolate(self):
         """Verify that our isolate/rejoin facilities actually work"""
         with firewall_blocked(self.redpanda.nodes, self._get_s3_endpoint_ip()):
             self.kafka_tools.produce(self.topic, 10000, 1024)
             time.sleep(10)  # can't busy wait here
-            self._check_bucket_is_emtpy()
+
+            # Topic manifest can be present in the bucket because topic is created before
+            # firewall is blocked. No segments or partition manifest should be present.
+            topic_manifest_id = "d0000000/meta/kafka/panda-topic/topic_manifest.json"
+            objects = self.s3_client.list_objects(self.s3_bucket_name)
+            keys = [x.Key for x in objects]
+            self.logger.debug(f"keys: {keys}")
```

Comment:
nit: `keys: ` is not a very descriptive log message (although it makes sense in the context of the code). Something like `objects in the bucket: ` is better.

Reply:
Just deleted this log statement since `keys` will be printed if the assertion fires.